lib/Distribution/Sle/15_current.pm

```diff
@@ -24,6 +24,7 @@ use Installation::License::Sle::LicenseAgreementController;
 use Installation::License::Sle::Firstboot::LicenseAgreementController;
 use Installation::ProductSelection::ProductSelectionController;
 use Installation::Registration::RegistrationController;
+use Installation::EncryptedVolume::EncryptedVolumeController;
```

Comment:
this is not SLE specific, even if we don't have a job there (don't we, right?) we should add it for TW distri.

Reply:
Wouldn't it make more sense to add it there once we add a relevant job for TW? With the same logic, should I add it for Leap too?

Reply:
TW in our distribution pattern is the default, we put in the other products when there is some exception or overwrite. As this logic is not specific (let's assume that the screen doesn't change much there) should go to Factory (TW) let's say and we can use it without changes when there will be some openSUSE job. So, in other words we should not put it in SLE or Leap, and put this code directly in TW.

Reply:
I didn't know. Thanks for the info. Done.